LDM-732.tex

```diff
@@ -60,35 +55,22 @@ \section{Introduction}\label{sec:intro}
-use the network. To be specific, this plan governs tests of the network only up to ISO OSI Layer
-\url{https://en.wikipedia.org/wiki/OSI_model}.
+This document pertains only to the verification of the network infrastructure, not the applications and services that use the network, specifically, up to \href{https://en.wikipedia.org/wiki/OSI_model}{OSI Layer} 3.
 
-As such, all of the tests governed by this plan and defined in the
-\href{https://jira.lsstcorp.org/projects/LVV/}{LSST Verification and Validation JIRA Project (LVV)}
-are defined as Lower Level (LL) in the System Engineering test hierarchy. LL
-corresponds to Data Management Subsystem Integration. Where appropriate, additional
-comments regarding Same Level (SL) which corresponds to LSST System Integration, and Higher
-Level (HL) which corresponds to LSST Commissioning, are called out in the Verification
-Elements.
+As such, all of the Verification Elements governed by this document and defined in the \href{https://jira.lsstcorp.org/projects/LVV/}{LSST Verification and Validation (LVV)} Jira project are Lower Level (LL) in the System Engineering test hierarchy.
 
-Note that significant testing of the networks occurs prior to subsystem and system integration,
-i.e. prior to verification, as documented in document-14789 LSST LHN End-to-End Plan and
-associated test documentation
-(see \href{https://docushare.lsstcorp.org/docushare/dsweb/View/Collection-3758}{Collection-3758}).
+LL corresponds to Data Management Subsystem Integration.
+Where appropriate, additional comments regarding Same Level (SL) which corresponds to LSST System Integration, and Higher Level (HL) which corresponds to LSST Commissioning, are called out in the Verification Elements.
 
-Finally, note that one significant network, the Summit Network, is not a DM deliverable and as
-such is not contained within this plan. As Telescope and Site deliverable, the Summit Network
-is covered by the Telescope and Site V\&V plans.
+Note that significant testing of the networks occurs prior to subsystem and system integration, i.e.\  prior to verification, as discussed in \href{https://ls.st/document-14789}{LSST LHN End-to-End Plan} and associated documentation in \href{https://docushare.lsstcorp.org/docushare/dsweb/View/Collection-3758}{Collection-3758}.
 
 \subsection{Specification Flow-down}\label{sec:sepcflowd}
 
 \subsubsection{Data Management Subsystem Requirements Flow-down}\label{sec:dmreqflowd}
 
-The Data Management Subsystem Requirements (LSE-61, aka DMSR) drive the LSST Observatory Network Design
-for all segments except the Summit Network (see above).  The DMSR sections that directly drive the
-VNVD are listed here for convenience. These DMSR sections contain traceable network requirements as
-documented in the LSST V\&V JIRA Project (LVV) Verification Elements:
+The Data Management Subsystem Requirements (\citeds{LSE-61}, aka DMSR) drive the LSST Observatory Network Design.
```

Comment:
The Data Management Subsystem Requirements (DMSR) (\citeds{LSE-61},

Reply:
???

Reply:
Done.